src/main/resources/static/js/ButtonDisabler.js

```diff
@@ -0,0 +1,10 @@
+document.addEventListener("DOMContentLoaded", function () {
+    const examNumberInput = document.getElementById("examNumber");
+    const submitButton = document.getElementById("submitButton");
+    function updateButtonState() {
+        const examNumber = examNumberInput.value;
+        submitButton.disabled = examNumber.length <= 4 || examNumber === "";
```

Comment:
Only check for empty

Reply:
Ive have changed it to only check for empty and made the test so they test exactly that